src/components/PeriodDimension/PeriodTransfer.js

```diff
@@ -162,35 +177,40 @@ const PeriodTransfer = ({
     return (
         <Transfer
             onChange={({ selected }) => {
-                const formattedItems = selected.map((id) => ({
-                    id,
-                    name: [...allPeriods, ...selectedPeriods].find(
+                const formattedItems = selected.map((id) => {
+                    const matchingItem = [...allPeriods, ...selectedItems].find(
                         (item) => item.id === id
-                    ).name,
-                }))
-                setSelectedPeriods(formattedItems)
+                    )
+
+                    return {
+                        id,
+                        name: matchingItem.name,
+                        isActive: matchingItem.isActive,
+                    }
+                })
                 onSelect(formattedItems)
             }}
-            selected={selectedPeriods.map((period) => period.id)}
+            selected={selectedItems.map((period) => period.id)}
             leftHeader={renderLeftHeader()}
             enableOrderChange
             height={TRANSFER_HEIGHT}
             optionsWidth={TRANSFER_OPTIONS_WIDTH}
             selectedWidth={TRANSFER_SELECTED_WIDTH}
             selectedEmptyComponent={renderEmptySelection()}
-            rightHeader={renderRightHeader()}
+            rightHeader={<RightHeader infoText={infoBoxMessage} />}
             rightFooter={rightFooter}
-            options={[...allPeriods, ...selectedPeriods].map(
-                ({ id, name }) => ({
-                    label: name,
-                    value: id,
-                })
-            )}
+            options={[...allPeriods, ...selectedItems].map(({ id, name }) => ({
+                label: name,
+                value: id,
+            }))}
             renderOption={(props) => (
                 <TransferOption
+                    /* eslint-disable react/prop-types */
```

Comment:
use `// eslint-disable-next-line` if it's only for a single line?

Reply:
or avoid it altogether with this?
```
renderOption={({ value, ...props }) => (
    <TransferOption
        {...props}
        active={isActive(value)}
        icon={PeriodIcon}
        dataTest={`${dataTest}-transfer-option`}
    />
)}
```

Reply:
Fixed by separating `value` from `props`.